pkg/block/local/adapter.go

```diff
@@ -552,15 +552,15 @@ func (l *Adapter) GetStorageNamespaceInfo(string) *block.StorageNamespaceInfo {
 	return &info
 }
 
-func (l *Adapter) ResolveNamespace(_, storageNamespace, key string, identifierType block.IdentifierType) (block.QualifiedKey, error) {
+func (l *Adapter) ResolveNamespace(storageID, storageNamespace, key string, identifierType block.IdentifierType) (block.QualifiedKey, error) {
```

Comment:
Not sure - why do we need this change?

Reply:
Adding this to the ObjectPointer initialized on line 562.

Even if it's not used by `extractParamsFromObj` at the moment, I rather add it for completeness and for future changes.